pandas/tests/tslibs/test_api.py

```diff
@@ -6,7 +6,8 @@
 
 def test_namespace():
 
-    submodules = ['ccalendar',
+    submodules = ['_timestamp',
```

Comment:
@ArtificialQualia  ``_timestamp`` is a little odd for our naming scheme as this entire set of modules is already private. maybe ``c_timestamp.pyx`` ?

cc @jbrockmendel